### PR TITLE
#3668 - Send the invalid beta user immediately to login page

### PR DESCRIPTION
**Pull request by andrepestana-aot**
The invalid beta user was being sent to the dashboard before being redirected to the login page with an invalid beta user message.

### CODE REVIEW

File: sources/packages/web/src/services/AuthService.ts [hunk 1]
```diff
@@ -267,6 +267,13 @@ export class AuthService {
         break;
       }
       case ClientIdType.Student: {
+        // Send the user to login page with invalid beta user message instead of dashboard.
+        this.priorityRedirect = {
+          name: StudentRoutesConst.INVALID_BETA_USER,
+        };
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/services/AuthService.ts:273]: To do the specific redirect to `INVALID_BETA_USER` should we move this inside the `if (options?.invalidBetaUser)`?

Comment (guru-aot) [reply sources/packages/web/src/services/AuthService.ts:273]: yes i had the same question.

Comment (andrepestana-aot) [reply sources/packages/web/src/services/AuthService.ts]: Yes, you are right. Changed. Thanks!

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @andrepestana-aot

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍

**Pull request merged**